Bottom: af7ef0d7df2b90736df3710d13e2ab0a540e8bdb
Top:    6dd19b7852c38403687005f3c4d5e67b759aeb9a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-03 00:16:54 +0100

Cure crosshair lagging on screen

FIXME: SHOULD NOT RE-WRITE THE GTK HID


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 9272004..8293b0c 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -684,9 +684,7 @@ ghid_port_window_motion_cb (GtkWidget * widget,
   queue_tooltip_update (out);
 #endif
 
-  ghid_show_crosshair (FALSE);
-  ghid_show_crosshair (TRUE);
-  if (moved && have_crosshair_attachments ())
+  if (moved)
     ghid_draw_area_update (gport, NULL);
   return FALSE;
 }
